components/Tabs/src/TabBarView/MDCTabBarView.m

```diff
@@ -99,32 +87,88 @@ - (void)setSelectedItem:(UITabBarItem *)selectedItem {
   _selectedItem = selectedItem;
 }
 
-#pragma mark - UIView
```

Comment:
#pragma mark - Private API should be added for the following functions.

Reply:
Fixed.